examples/node-api-sock-path/README.md

```diff
@@ -0,0 +1,15 @@
+# Node.js API - Simple
```

Comment:
Please remove all `examples` as we are don't intend to maintain them in the future (#1469)